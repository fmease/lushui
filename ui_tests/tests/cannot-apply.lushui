;;; TEST fail check

Abomination: Type = 0 1
