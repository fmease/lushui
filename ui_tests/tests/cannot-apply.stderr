error[E031]: expected type `_ -> _`, got type `crate.core.nat.Nat`
 --> ${DIRECTORY}/cannot-apply.lushui:6:21
  |
6 | Abomination: Type = 0 1
  |                     ^ has wrong type
  |
 --> ${DIRECTORY}/cannot-apply.lushui:6:23
  |
6 | Abomination: Type = 0 1
  |                       - applied to this
  |

error: aborting due to previous error
 note: the error E031 has a detailed explanation
 help: run `lushui explain E031` to view it

